index.html

```diff
@@ -305,6 +305,54 @@ <h3>Skills</h3>
       </section>
     </main>
     <footer class="get-touch grid grid-col-1">
+      <div>
+        <p class="get-text">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </div>
+      <div>
+        <form
+          class="flex-col contact-form"
+          action="https://formspree.io/f/xbjewokr"
+          method="POST"
+        >
+          <ul class="list-form">
+            <li>
+              <input
+                type="text"
+                name="text"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="Email"
+                name="Email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="textarea"
+                placeholder="Write me something..."
+                id="feedback"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li class="btn-form">
+              <button type="submit" class="project-btn resume-btn">
+                Get in touch
+              </button>
+            </li>
+          </ul>
```

Comment:
- `[OPTIONAL ]` I have impressive work so far.  Kindly consider removing the `ul` and `li` tags you have around the inputs as they're not needed in creating a form input. This will reduce code redundancy and make your code more readable.

Reply:
ok thanks a lot